cmd/sriov-network-operator-config-cleanup/cleanup_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"context"
+
+	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/test/util"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	testNamespace            string = "sriov-network-operator"
+	defaultSriovOperatorSpec        = sriovnetworkv1.SriovOperatorConfigSpec{
+		EnableInjector:        true,
+		EnableOperatorWebhook: true,
+		LogLevel:              2,
+		FeatureGates:          nil,
+	}
+)
+
+var _ = Describe("cleanup", Ordered, func() {
+
+	defaultSriovOperatorConfig := &sriovnetworkv1.SriovOperatorConfig{
+		ObjectMeta: metav1.ObjectMeta{Name: "default", Namespace: testNamespace},
+		Spec:       defaultSriovOperatorSpec,
+	}
+
+	BeforeEach(func() {
+		Expect(k8sClient.Create(context.Background(), defaultSriovOperatorConfig)).NotTo(HaveOccurred())
```

Comment:
set a finalizer on it as well like in real case ?

Reply:
Ok. For this case I'll need to add mocked controller to verify that config object is deleted correctly